synapse/storage/databases/main/end_to_end_keys.py

```diff
@@ -361,6 +361,14 @@ def _get_e2e_device_keys_txn(
                     display_name, db_to_json(key_json) if key_json else None
                 )
 
+        for query_clause, query_params in zip(query_clauses, query_params_list):
+            await self.db_pool.runInteraction(
+                "_get_e2e_device_keys",
```

Comment:
I'm not sure if you purposefully used a new transaction description or not (it used to not have the `_` in front of it I think).

Reply:
Oh, doesn't really matter. Though unintended consequence of this means we're not going to confuse the two types of transactions in the graphs

Reply:
> we're not going to confuse the two types of transactions in the graphs

Yeah sometimes I feel like it is desirable to re-use it, even if it isn't apples-to-apples. I guess that's more important with cache names, maybe. 🤷